src/postgres/interface.c

```diff
@@ -758,6 +747,21 @@ pgXactPath(unsigned int pgVersion)
 /**********************************************************************************************************************************/
 #ifdef DEBUG
 
+unsigned int
+pgCatalogTestVersion(unsigned int pgVersion)
```

Comment:
So I suppose there was a reason why this is here and not solely in the test harness?

Reply:
The main reason is that the data structure and search function to make this work already exist here. At some point we'll move this into test but that will require being able to interface with the per-version PG code. Rather than make that split for this one function I think it makes sense to do it all together later.